sources/packages/web/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM artifacts.developer.gov.bc.ca/redhat-access-docker-remote/ubi8/nodejs-16:1-5 AS builder
+FROM artifacts.developer.gov.bc.ca/redhat-access-docker-remote/ubi8/nodejs-18:1-71.1697652955 AS builder
```

Comment:
Shouldn't we change this to nginx?

Reply:
No, this is just a intermediary image used to build the application that has `npm` package. if we use a nginx image will need to install node/npm:
![image](https://github.com/bcgov/SIMS/assets/78114138/26bb6d0b-3eed-4c64-b3fd-00a12b40778e)

The image that will run the application is a nginx one(line 15).

Reply:
Ah I see. I remember that from the NGINX PR now.